pom.xml

```diff
@@ -28,14 +28,14 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.48</version>
+        <version>3.54</version>
         <relativePath />
     </parent>
     <artifactId>pipeline-utility-steps</artifactId>
     <packaging>hpi</packaging>
     <version>${revision}${changelist}</version>
     <name>Pipeline Utility Steps</name>
-    <url>https://wiki.jenkins.io/display/JENKINS/Pipeline+Utility+Steps+Plugin</url>
+    <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
```

Comment:
To do this change you must first move the docs from the wiki to the readme. But I also don't like putting user docs in the root readme, that file is more imho for visitors of the repo itself. So the docs should be put in for example `docs/README.md`

Reply:
> To do this change you must first move the docs from the wiki to the readme.

I have no interest in doing this, so I will revert the change.